.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 jobs:
   verify_build:
-    working_directory: ~/repo
+    working_directory: ~/repo/serverless
```

Comment:
Looking for a better long term solution where circle ci only runs certain tests if code in a certain directory changes, so in the future this repo can be used for other macros, but I wanted to get this PR out first